### PR TITLE
jack-example-tools: init at 4

### PR DESCRIPTION
**Pull request by pennae**
## Description of changes

jack2 dropped its tools from the main package recently. they now live here. examples are included too as previously, and unlike previously we also include `zalsa` plugins. they're tiny though, adding them won't hurt anything.

darwin is marked as broken because we have no clue where to even start with darwin, not because it's actually known to not work.

fixes  #253188

@cillianderoiste @matthiasbeyer  @risicle

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Janik-Haag** [comment]
Result of `nixpkgs-review pr 253299` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>jack-example-tools</li>
  </ul>
</details>

**Review by Janik-Haag — APPROVED** [review]
LGTM; tools seem to work fine code is nice.

File: pkgs/misc/jackaudio/tools.nix [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  # no tests defined, but prepare for some in the future.
+  doCheck = true;
+
+  meta = with lib; {
+    description = "Official examples and tools from the JACK project";
+    homepage = "https://jackaudio.org";
+    license = licenses.gpl2Plus;
+    platforms = platforms.unix;
+    broken = stdenv.isDarwin;
```
Comment (Janik-Haag) [inline pkgs/misc/jackaudio/tools.nix:55]: you might want to add the `meta.sourceProvenance` attribute

Comment (Janik-Haag) [reply pkgs/misc/jackaudio/tools.nix:55]: fair enough

**Comment by matthiasbeyer** [comment]
Quite a huge number of rebuilds... should this go to staging?

**Comment by Janik-Haag** [comment]
That was just initially as you can see above ofborg dropped the labels again. It also confused me the first time I looked at it.

**Comment by Janik-Haag** [comment]
cc @NixOS/darwin-maintainers

**Comment by rvl** [comment]
@pennae, did you not search for existing PRs before opening this?

I had already created #243378 months before. It makes me wonder what's the point in putting in hours of effort building and testing if the PR will just be ignored.

**Comment by n8henrie** [comment]
While having people do their due diligence on existing work prior to embarking on a PR seems like the best solution, I wonder if there's any room here for another bot that might search existing open PRs and reply with a list of them. Unfortunately I expect people are probably not opening PRs until they have *already* put in significant effort.

**Comment by rvl** [comment]
Yes, sorry for the harsh words @pennae. It's not your fault.

I believe you are correct that the PR review tools on github are not fit for task.

**Pull request merged**